File: Clientside/Admin/src/models/CreateVariantOrderDto.ts
import InventoryInfoDto from "./InventoryInfoDto";
import ProductDto from "./ProductDto";

interface CreateVariantOrderDto {
    sizeId: number,
    variantId: number,
    orderedItemsTotal: number,
    product?: ProductDto
    inventoryInfos?: InventoryInfoDto[]
}


export default CreateVariantOrderDto

File: Serverside/Admin/Api/Dtos/Order/CreateVariantOrderDto.cs
using System.ComponentModel.DataAnnotations;

namespace FNO.Admin.AdminApi.Dtos.Order
{
    public class CreateVariantOrderDto
    {
        [Required]
        public int SizeId { get; set; }
        [Required]
        public int VariantId { get; set; }
        [Required]
        public int OrderedItemsTotal { get; set; }
    }
}
